cunumeric/runtime.py

```diff
@@ -39,33 +39,18 @@
 from .eager import EagerArray
 from .thunk import NumPyThunk
 from .types import NdShape
-from .utils import calculate_volume, find_last_user_stacklevel, get_arg_dtype
+from .utils import (
+    _SUPPORTED_DTYPES,
```

Comment:
any reason not to just make it "public"?

Reply:
Not really! d2a2786ee7f8af242bb1d606ec937ac7f25057b0